spec/support/store.rb

```diff
@@ -0,0 +1,16 @@
+class Store
```

Comment:
`module` instead of `class`

`Store` is a really generic name, some other gem could export a class/module with the same name and in that case we'll face some problems here